taxi/urls.py

```diff
@@ -1,9 +1,24 @@
 from django.urls import path
 
-from .views import index
+from .views import (index,
+                    ManufacturerListView,
+                    CarListView,
+                    DriverListView,
+                    CarDetailView,
+                    DriverDetailView
+                    )
 
 urlpatterns = [
     path("", index, name="index"),
+    path("manufacturers/",
+         ManufacturerListView.as_view(),
+         name="manufacturer-list"
+         ),
+    path("cars/", CarListView.as_view(), name="car-list"),
+    path("cars/<int:pk>", CarDetailView.as_view(), name="car-detail"),
+    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/<int:pk>", DriverDetailView.as_view(), name="driver-detail"),
```

Comment:
Place the closing slash at the end of the path where it is absent
```suggestion
    path("drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"),
```